MAINTAINERS.md

```diff
@@ -2,13 +2,14 @@
 
 ## Maintainers
 
-| Maintainer         | GitHub ID                                         |                              Affiliation |
-| ------------------ | ------------------------------------------------- | ---------------------------------------: |
-| Antonio Gámez Díaz | [antgamdia](https://github.com/antgamdia)         | [VMware](https://www.github.com/vmware/) |
-| Greg Fichtenholtz  | [gfichtenholt](https://github.com/gfichtenholt)   | [VMware](https://www.github.com/vmware/) |
-| Michael Nelson     | [absoludity](https://github.com/absoludity)       | [VMware](https://www.github.com/vmware/) |
-| Pepe Baena         | [ppbaena](https://github.com/ppbaena)             | [VMware](https://www.github.com/vmware/) |
-| Rafa Castelblanque | [castelblanque](https://github.com/castelblanque) | [VMware](https://www.github.com/vmware/) |
+| Maintainer         | GitHub ID                                           |                              Affiliation |
```

Comment:
These changes have been inadvertently commited perhaps?

Reply:
Of course. Thanks for the heads up